add(epp): applications-docker-ssh (#2504)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-docker-ssh.md

and in the [following chapter](/docs/monitoring/discovery/services-discovery/#discovery-rules).

### Collected metrics & status

Here is the list of services for this connector, detailing all metrics linked to each service.

<Tabs groupId="sync">
<TabItem value="Container-Status" label="Container-Status">

| Metric name       | Unit  |
|:------------------|:------|
| containers status |       |

</TabItem>
<TabItem value="Container-Usage" label="Container-Usage">

| Metric name                                                     | Unit  |
|:----------------------------------------------------------------|:------|
| *container_name*#container.cpu.utilization.percentage           | %     |
| *container_name*#container.memory.usage.bytes                   | B     |
| *container_name*#container.disk.throughput.read.bytespersecond  | B/s   |
| *container_name*#container.disk.throughput.write.bytespersecond | B/s   |
| *container_name*#container.traffic.in.bitspersecond             | b/s   |
| *container_name*#container.traffic.out.bitspersecond            | b/s   |

</TabItem>
</Tabs>

## Prerequisites

### SSH configuration

> A user is required to query the resource by SSH. There is no need for root or sudo

Reviewer: ```suggestion
A user is required to query the resource using SSH. There is no need for root or sudo
```

privileges. There are two possible ways to log in through SSH, either by

> exchanging the SSH key from **centreon-engine** user to the target resource, or by

Reviewer: ```suggestion
copying the SSH key of the **centreon-engine** user to the target resource, or by
```

```bash
apt install centreon-plugin-applications-docker-ssh
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">

```bash
yum install centreon-plugin-Applications-Docker-Ssh
```

</TabItem>
</Tabs>

## Using the monitoring connector

### Using a host template provided by the connector

1. Log into Centreon and add a new host through **Configuration > Hosts**.
2. Fill the **Name**, **Alias** & **IP Address/DNS** fields according to your ressource settings.
3. Apply the **App-Docker-SSH-custom** template to the host. A list of macros appears. Macros allow you to define how the connector will connect to the resource, and to customize the connector's behavior.
4. Fill in the macros you want. Some macros are mandatory.

| Macro           | Description                                                                                                                                                         | Default value     | Mandatory   |
|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | SSHUSERNAME     | Define the user name to log in to the host                                                                                                                          |                   |             |

Reviewer: mandatory?

Reviewer 2: Nope

> | SSHPASSWORD     | Define the password associated with the user name. Cannot be used with the sshcli backend. Warning: using a password is not recommended. Use --ssh-priv-key instead |                   |             |

Reviewer: mandatory? why isn't there a line for --ssh-priv-key?

Reviewer 2: Nope
